src/webrtc/call.ts

```diff
@@ -1019,7 +1032,14 @@ export class MatrixCall extends EventEmitter {
             return;
         }
 
-        const prevOnHold = this.isLocalOnHold();
+        const prevLocalOnHold = this.isLocalOnHold();
+
+        if (description.type === 'answer') {
+            // whenever we get an answer back, clear the flag we set whilst trying to un-hold
+            // the other party: the state of the channels now reflects reality
+            //nope, this doesnt work either
```

Comment:
Hmm, what does work then? 😜

Reply:
haha, whoops - the reason it didn't work was because I wasn't checking the flag in the right place, and of course I forgot to remove the comment!